test/delete.js

```diff
@@ -0,0 +1,58 @@
+const test = require('tape')
+const ssbKeys = require('ssb-keys')
+const path = require('path')
+const rimraf = require('rimraf')
+const mkdirp = require('mkdirp')
+const validate = require('ssb-validate')
+const pull = require('pull-stream')
+const SecretStack = require('secret-stack')
+const caps = require('ssb-caps')
+
+const { author, type } = require('../operators')
+
+const dir = '/tmp/ssb-db2-delete'
+
+rimraf.sync(dir)
+mkdirp.sync(dir)
+
+const keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+
+const sbot = SecretStack({ appKey: caps.shs })
+  .use(require('../'))
+  .use(require('../compat/ebt'))
+  .call(null, {
+    keys,
+    path: dir,
+  })
+const db = sbot.db
+
+test('index, delete, query', (t) => {
+  const post = { type: 'post', text: 'Testing!' }
+  const post2 = { type: 'post', text: 'Testing 2!' }
+
+  db.publish(post, (err, postMsg) => {
+    t.error(err, 'no err')
+
+    db.log.onDrain(() => {
```

Comment:
Just a heads up that this might need to be rebased with #85 so that it's `db.getLog()` etc